src/index.html

```diff
@@ -11,7 +11,150 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      autocomplete="off"
+    >
+      <fieldset>
+        <legend>Personal information</legend>
+        <label for="surname">Surname: </label>
+        <input 
+          type="text" 
+          id="surname" 
+          name="surname" 
+          required
+        >
+        <br>
+        <label for="name">Name:</label>
```

Comment:
It's hard to read now. Why do you need the br tags everywhere?
```suggestion
        >

        <label for="name">Name:</label>
```